tools/pd-ctl/pdctl/command/store_command.go

```diff
@@ -377,7 +377,7 @@ func storeLimitCommandFunc(cmd *cobra.Command, args []string) {
 	case 2, 3:
 		rate, err := strconv.ParseFloat(args[1], 64)
 		if err != nil || rate < 0 {
-			cmd.Println("rate should be a number that >= 0.")
+			cmd.Println("rate should be a number that > 0.")
```

Comment:
do we need to change `if rate < 0` to `if rate <= 0`?

Reply:
Oh, yes!